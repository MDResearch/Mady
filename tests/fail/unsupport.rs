use mady::prelude::*;

#[grad(f64, f64)]
fn unsupport_ops(a: f64, b: f64) -> f64 {
    a >> b ^ b << a
}

fn main() {}
