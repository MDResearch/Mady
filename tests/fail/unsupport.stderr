error: unsupported syntax
 --> tests/fail/unsupport.rs:5:7
  |
5 |     a >> b ^ b << a
  |       ^^
